server/test/unit/apiTest/authorizeTest.js

```diff
@@ -44,90 +32,90 @@ describe('api.js unit tests: authorize', () => {
   describe('when a request provides the primary PA API key in the query string of the request', () => {
     beforeEach(async () => {
       // create fake request object that implements the necessary members
-      const req = mockRequest({
+      const req = {
         query: { braveKey: helpers.getEnvVar('PA_API_KEY_PRIMARY') },
         body: {}, // empty
         path: 'test_path',
-      })
-      this.res = mockResponse()
+      }
+      this.res = mockResponse(sandbox)
```

Comment:
Thanks for passing `sandbox` instead of `sinon` here! That will definitely help with cleanup.